locales/en-US/app.ftl

```diff
@@ -1,8 +1,48 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 ### Localization for the App UI of Profiler
 
-# Naming convention for l10n IDs: "module-ComponentName--string-summary".
+# Naming convention for l10n IDs: "ComponentName--string-summary".
 # This allows us to minimize the risk of conflicting IDs throughout the app.
-# Please sort alphabetically by (module name, component name). For each module,
+# Please sort alphabetically by (component name).
 # keep strings in order of appearance.
 
-intro = Web app for Firefox performance analysis
+## The following feature names must be treated as a brand. They cannot be translated
+-brand-name = Firefox
+-profiler-brand-name = Firefox Profiler
+
+# Home page translations
+AppHeader--app-header-slogan= { -profiler-brand-name }
+AppHeader--app-header-subtext = {" "}&mdash; Web app for Firefox performance analysis
```

Comment:
This should be localizable, and you should always use the UTF character instead of HTML entities.

Is there a technical reason that prevents exposing this as a single string? (I know pretty much nothing about React)

```
<span>{ -profiler-brand-name }</span> — Web app for Firefox performance analysis
```

That's how it should be done to avoid any potential localization issue.

Reply:
I tried to combine these two text strings but the problem is these are wrapped with two different elements and have two different class style. So I couldn't wrap with `Localied` because it's showing the error , `<Localized/> expected to receive a single React node child`.

Reply:
Bear with me, I have never looked at React code before. Is there no way to work around it? What would prevent something like this from working?

```
<span className="appHeaderSlogan">
    <Localized 
        id="AppHeader--app-header"
        elems={{
            header: <InnerNavigationLink dataSource="none" className="appHeaderLink"></InnerNavigationLink>,
            subheader: <span className="appHeaderSubtext"></span>
        }}
    >
        <header>Firefox Profiler</header> – <subheader>Web app for Firefox performance analysis</subheader>
    </Localized>
</span>
```